Documentation/FAQs.md

```diff
@@ -15,6 +16,20 @@ The file **launch.json** will now be open for editing with a new configuration.
 
 See the [**Documentation/Debugger**](https://github.com/Microsoft/vscode-cpptools/tree/master/Documentation/Debugger) folder in this repository for more in-depth documentation on how to configure the debugger.
 
+## What is the .vscode/ipch folder?
```

Comment:
I think this should be split into 2 different questions that answer: 
1. How do I disable the ipch folder?
2. How do I move the ipch folder?
Which is similar to the sections describing the settings.